projectEuler/problem5.js

```diff
@@ -0,0 +1,28 @@
+// prompt: find the smallest positive number that is evenly divisible by all of the numbers from 1 to 20
+
+// helper function for checking if integer is divisible by all numbers from 1 to 20
+function isDivisible(int) {
+  var count = 0;
+  for (let i = 1; i <= 20; i++) {
+    if (int % i === 0) {
+      count++;
+    }
+  }
+  if (count === 20) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You can drop the `else` block here and just `return false` by default. I.e:

````
if (count === 20) {
    return true;
}

return false;
````